src/ui/components/Comments/TranscriptComments/CommentSource.tsx

```diff
@@ -41,21 +41,35 @@ function CommentSource({
     selectLocation(context, sourceLocation);
   };
 
+  if (!sourceLocation) {
+    return null;
+  }
+
+  let sourceContent: React.ReactNode = null;
+  // We may not have sources yet, and once we have sources we still have to calculate
+  // the "labels" from the source texts.  Show loading indicators until ready.
+  if (labels) {
+    sourceContent = (
+      <div
+        className="cm-s-mozilla overflow-hidden whitespace-pre font-mono text-xs"
+        style={{ fontSize: "11px" }}
+        dangerouslySetInnerHTML={{ __html: labels?.secondary || "" }}
+      />
+    );
+  } else {
+    sourceContent = <div className="whitespace-pre font-mono text-xs italic">Loading...</div>;
```

Comment:
It'd be nice if we had a CSS treatment for pulsing the background color or creating some other sort of shimmer
https://tailwindcss.com/docs/animation#pulse

Reply:
Visually, I think we should avoid the combination of monospaced font and _italic_ style.

Maybe our loader should just be a shimmer or a line or something.